website/docs/r/cms_alarm.html.markdown

```diff
@@ -55,6 +56,7 @@ The following arguments are supported:
 * `silence_time` - Notification silence period in the alarm state, in seconds. Valid value range: [300, 86400]. Default to 86400
 * `notify_type` - Notification type. Valid value [0, 1]. The value 0 indicates TradeManager+email, and the value 1 indicates that TradeManager+email+SMS
 * `enabled` - Whether to enable alarm rule. Default to true.
+* `webhook`- The webhook that should be called when the alarm is triggered. Currently, only http protocol is supported. Default is empty string.
```

Comment:
Please add a tag for the `webhook - (Optional, Available in 1.46.0+)`

Reply:
done